### DISCUSSION TITLE
Hit 0, miss keeps growing

### DISCUSSION THREAD

**Discussion by Sparklehorse001** [ccache/ccache#930] [Support]
ccache version 3.7.6

if type -p ccache >/dev/null 2>&1; then
  export rootDir=$pwd
  export CCACHE_MAXSIZE=10G
  export CCACHE_CPP2=true
  export CCACHE_HARDLINK=true
  export CCACHE_SLOPPINESS=file_macro,time_macros,include_file_mtime,include_file_ctime,file_stat_matches
  export CCACHE_COMPILERCHECK=content
  exec ccache /usr/bin/clang "$@"
else
  exec clang "$@"
fi

result：
 Hit 0， 
Miss  keeps growing

**Comment by jrosdahl** [comment]
Hi there,

I'm afraid your question is far too vague to be able to answer. Here are some hints on what you can do:

* Read tips about [ccache cache debugging](https://ccache.dev/manual/3.7.6.html#_cache_debugging).
* Read more general tips about [how to get help from maintainers](https://www.snoyman.com/blog/2017/10/effective-ways-help-from-maintainers/).
